### PR TITLE
task 9 and refactoring

### PR DESCRIPTION
**Pull request by katmiah**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -292,7 +292,34 @@ describe("DELETE /api/comments/:comment_id", () => {
...
-      expect(body.message).toBe("Bad request.")
+      expect(body.message).toBe("Invalid request.")
     })
   })
-})
\ No newline at end of file
+})
+
+describe("GET /api/users", () => {
+  test("200: Responds with an array of objects with user properties", () => {
```
Comment (nc-aimee-muter) [inline __tests__/app.test.js:301]: Lovely stuff!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -292,7 +292,34 @@ describe("DELETE /api/comments/:comment_id", () => {
...
+      users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String),
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  })
+  test("404: Responds with error message if endpoint is invalid", () => {
```
Comment (nc-aimee-muter) [inline __tests__/app.test.js:317]: You only need one of these tests in your whole app and none of them relate to any particular endpoint (so this shouldn't be in this describe block).
